docs/getting-started/launching-bpfman.md

```diff
@@ -0,0 +1,137 @@
+# Launching bpfman
+
+The most basic way to deploy bpfman is to run it directly on a host system.
+First `bpfman` needs to be built and then started.
```

Comment:
We don't have to change this all now, but wouldn't it be better to point the user at running release as a first option?  Then say, or you can build it yourself...

Reply:
@anfredette Added a blurb to the next section (`Build bpfman`) linking to Release or RPMs installs. Let me know if that helps. Bit of a circular link pointing, because Release page points back to the following section (`Start bpfman-rpc`) for more details on modes to run in.

Reply:
> @anfredette Added a blurb to the next section (`Build bpfman`) linking to Release or RPMs installs. Let me know if that helps. Bit of a circular link pointing, because Release page points back to the following section (`Start bpfman-rpc`) for more details on modes to run in.

It's fine with me as is.  I think that in the future, it would be good to start off with a whole section about using bpfman without any need to build anything.  E.g., by downloading and using released images, or installing from RPMs.  Then move into building it, and more advanced development topics.

This LGTM.  It's okay with me if you squash and then remove the hold.  Thanks!